src/templates/index.jade

```diff
@@ -8,6 +8,37 @@ html
         link(rel='stylesheet' type='text/css' href='public/style.css')
         script(src='public/head.js')
     body(itemtype='http://schema.org/EducationEvent', itemscope)
+        nav.top-bar(data-topbar role='navigation')
+            ul.title-area
+                li.name
+                    h1
```

Comment:
There is empty title. Because title area is important when page is opened on mobile.